subsys/bluetooth/scan.c

```diff
@@ -1248,6 +1248,11 @@ static bool adv_data_found(struct bt_data *data, void *user_data)
 		manufacturer_data_check(scan_control, data);
 		break;
 
+	case BT_DATA_LE_BT_DEVICE_ADDRESS:
+
```

Comment:
excessive empty lines

Reply:
This whole case should be removed. I had to add it during testing.